src/XPlot.Plotly/Main.fs

```diff
@@ -40,7 +41,15 @@ module Html =
         let file = sprintf "%s.html" pageId
         let path = Path.Combine(tempPath, file)
         File.WriteAllText(path, html)
-        System.Diagnostics.Process.Start(path) |> ignore
+        if RuntimeInformation.IsOSPlatform(OSPlatform.Windows) then
```

Comment:
Why is this not a match?

Reply:
`RuntimeInformation` only exposes API `IsOsPlatform` which is `OsPlatform -> bool`. Writing it with match ... with won't make it more simple.
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform?view=netframework-4.8